src/config/index.ts

```diff
@@ -0,0 +1,2 @@
+export const isAllowCreateAccount = false;
+export const isAllowSignIn = false;
```

Comment:
@jamessspanggg 
I think a better name would be `isCreateAccountAllowed` or `isCreateAccountVisible`.

Same for sign-in.

Reply:
yep changed